.secrets

```diff
@@ -0,0 +1,17 @@
+ops/ansible/playbooks-ccs/host_vars/builds.bfd-mgmt.cmscloud.local/vault.yml
```

Comment:
@ericbuckley did you do something programmatically (scan repo in some fashion) to compile this list or was it done manually? 

@keithdadkins just flagging you in case you notice any other files that should be included.